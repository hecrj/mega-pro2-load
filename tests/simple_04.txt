2
10
20

4
1 2 3 0 0 0 4 0 0

10 1 1
20 1 2
10 1 1
40 1 2

-3 1
-1 1 100
-6
