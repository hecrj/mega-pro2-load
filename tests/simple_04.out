Servidores ocupados
0
Peticion procesada y servidores
0 1
1
